about-page.css

```diff
@@ -0,0 +1,795 @@
+  .partnermobile-intro-border {
+    width: 2rem;
+    border-bottom: 2px solid rgb(239, 81, 60);
+    margin: 0 auto 1.5625rem;
+  }
+
+  .partnermobile-logo {
+    padding-bottom: 5.658%;
+    text-align: center;
+  }
+
+  .partnermobile-logo img {
+    width: 25%;
+    margin-left: 4%;
+    margin-right: 3%;
+  }
+
+  /* Partner Mobile Ends */
+
+  /* Rights Starts */
+  .rigts {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .rights-img-desktop {
+    display: none;
+  }
+
+  .rights-img-mobile {
+    width: 30%;
+    padding-top: 15px;
+    padding-bottom: 20px;
+    padding-left: 10px;
+  }
+
+  .rights-msg {
+    display: flex;
+    align-items: center;
+    width: 72%;
+    padding-left: 4.514%;
+    padding-top: 10px;
+    padding-bottom: 10px;
+    padding-right: 40px;
+  }
+
+  .rights-msg h4 {
+    color: #3b3e45;
+    font-size: 0.625rem;
+    font-weight: 700;
+  }
+
+  /* Rights Ends */
+}
+
+@media only screen and (min-width: 768px) {
+  .ham-popup {
+    display: none;
+  }
+
+  .partner-mobile {
+    display: none;
+  }
+
+  /* nav banner starts */
+  .nav-banner {
+    background: #3e3e3e;
+  }
+
+  .nav-banner-list {
+    display: flex;
+    justify-content: end;
+    padding-right: 13%;
+    margin-block-end: 0;
+    padding-bottom: 0.42rem;
+  }
+
+  .nav-banner-item {
+    list-style-type: none;
+    color: #fff;
+    padding-left: 1.26rem;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 0.79rem;
+  }
+
+  /* nav banner ends */
+
+  /* navbar starts */
+  .hambar {
+    display: none;
+  }
+
+  .my-navbar-intro {
+    box-shadow: 0 15px 10px -15px #111;
+    padding-top: 13px;
+  }
+
+  .navbar {
+    padding-top: 0.84rem;
+  }
+
+  #no-wrap {
+    flex-wrap: nowrap;
+  }
+
+  .navbar-brand {
+    padding-top: 0;
+    padding-left: 9%;
+  }
+
+  .navbar-brand img {
+    max-width: 60%;
+    padding-bottom: 13px;
+  }
+
+  .navbar-ul {
+    display: flex;
+  }
+
+  .navbar-item {
+    padding-right: 3rem;
+    padding-top: 1.6%;
+    list-style: none;
+  }
+
+  .navbar-link {
+    color: #868686;
+    font-weight: 800;
+    font-size: 0.96rem;
+    text-decoration: none;
+  }
+
+  .navbar-content {
+    display: flex;
+
+    /* flex-wrap: inherit; */
+    align-items: center;
+
+    /* flex-direction: row-reverse; */
+    justify-content: space-between;
+    padding: 0;
+  }
+
+  .navbar-ALD {
+    border: 5px solid #ec4327;
+    color: #ec4327;
+    font-weight: 800;
+    font-size: 0.96rem;
+    text-decoration: none;
+    padding: 15px;
+  }
+
+  .first-navbar-wrapper {
+    width: 125%;
+    padding-bottom: 10px;
+  }
+
+  /* navbar ends */
+
+  /* Summit Description Starts */
+  .summit-description {
+    background-image:
+      linear-gradient(
+        rgba(255, 255, 255, 0.6),
+        rgba(255, 255, 255, 0.6)
+      ),
+      url(./new-img/about-img.jpg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position-x: 50%;
+    margin-top: 0.2%;
+  }
```

Comment:
- [ ] This keeps getting closer but please make sure you use the proper color for the different paragraphs in your design as you seem to be missing some of them please check the screenshot below 👇
- Your design:
  ![image](https://github.com/Abbas-Aljuaifari/capstone-project-repo2/assets/104968236/558006b9-c13d-40fe-90ed-7bb48c1cb7cf)
- Behance design:
  ![image](https://github.com/Abbas-Aljuaifari/capstone-project-repo2/assets/104968236/9880b36f-be2c-4756-aa12-ce7044a3446d)